src/main/thrift/parquet.thrift

```diff
@@ -625,7 +631,11 @@ struct DataPageHeaderV2 {
   /** Number of NULL values, in this data page.
       Number of non-null = num_values - num_nulls which is also the number of values in the data section **/
   2: required i32 num_nulls
-  /** Number of rows in this data page. which means pages change on record boundaries (r = 0) **/
```

Comment:
I also propose expanding the `r = 0` terminology to be explicitly `repetition_level = 0` to match the conventions in the rest of the documentation